jax/_src/numpy/ufuncs.py

```diff
@@ -303,9 +303,10 @@ def right_shift(x1: ArrayLike, x2: ArrayLike, /) -> Array:
     np.issubdtype(x1.dtype, np.unsignedinteger) else lax.shift_right_arithmetic
   return lax_fn(x1, x2)
 
-@implements(getattr(np, "bitwise_right_shift", np.right_shift), module='numpy')
+
 @partial(jit, inline=True)
 def bitwise_right_shift(x1: ArrayLike, x2: ArrayLike, /) -> Array:
+  """Alias of :func:`jax.numpy.right_shift`."""
   x1, x2 = promote_args_numeric("bitwise_right_shift", x1, x2)
```

Comment:
Maybe we should `return right_shift(x1, x2)` for simplicity?

Reply:
agreed... is there any side-effects/consequence of having a technically different name passed to promote_args_numeric?

Reply:
It just means that if someone passes invalid values the error message will refer to the aliased function

Reply:
It seems the removed line: `@implements(getattr(np, "bitwise_right_shift", np.right_shift), module='numpy')` causes the CI test to look for np.bitwise_right_shift instead, which doesn't exist:

```
_______________ JaxNumpyOperatorTests.testShiftOpAgainstNumpy0 ________________
[gw5] linux -- Python 3.9.19 /opt/hostedtoolcache/Python/3.9.19/x64/bin/python
tests/lax_numpy_operators_test.py:657: in testShiftOpAgainstNumpy
    np_op = getattr(np, op.__name__)
/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/numpy/__init__.py:333: in __getattr__
    raise AttributeError("module {!r} has no attribute "
E   AttributeError: module 'numpy' has no attribute 'bitwise_right_shift'
=========================== short test summary info ============================
FAILED tests/lax_numpy_operators_test.py::JaxNumpyOperatorTests::testShiftOpAgainstNumpy0 - AttributeError: module 'numpy' has no attribute 'bitwise_right_shift'
========== 1 failed, 10655 passed, 17048 skipped in 399.33s (0:06:39) ==========
Error: Process completed with exit code 1.
```

I tried testing it locally:
```sh
python tests/lax_numpy_operators_test.py
```

but it's not showing locally:
```
Ran 1612 tests in 30.132s

OK (skipped=2)
```

It seems like we need to skip this test for aliases not defined in numpy. Would you happen to know how I can run this locally? or how we can cause the CI/CD to skip this test for this case?

Reply:
With pytest:

```sh
pytest tests/lax_numpy_operators_test.py
================================================================================== test session starts ===================================================================================
platform darwin -- Python 3.9.19, pytest-8.2.2, pluggy-1.5.0
rootdir: /Users/pisethk/my_jax
configfile: pyproject.toml
collected 1612 items                                                                                                                                                                     

tests/lax_numpy_operators_test.py ................................................................................................................................................ [  8%]
.................................................................................................................................................................................. [ 19%]
.................................................................................................................................................................................. [ 31%]
.................................................................................................................................................................................. [ 42%]
.................................................................................................................................................................................. [ 53%]
.................................................................................................................................................................................. [ 64%]
.................................................................................................................................................................................. [ 75%]
.................................................................................................................................................................................. [ 86%]
.................................................................................................................................................................................. [ 97%]
..............................ss............                                                                                                                                       [100%]

============================================================================ 1610 passed, 2 skipped in 37.00s ============================================================================
```